src/datatype/index.js

```diff
@@ -239,12 +239,17 @@ export class DataType extends TypedEmitter {
     return this.getByDocId(docId)
   }
 
-  /**
-   * @param {Parameters<import('drizzle-orm/better-sqlite3').BetterSQLite3Database['select']>[0]} fields
-   */
```

Comment:
This parameter (1) isn't in our type definitions (2) isn't used, so I removed it for simplicity.